Managers/FrameMan.cpp

```diff
@@ -805,29 +776,38 @@ namespace RTE {
 
 					if (SaveIndexedBitmap(fullFileName, scenePreviewDumpBuffer, m_Palette) == 0) {
```

Comment:
Why is this not swapped to png, just didn't get to it? Or is it actually and I'm dum and didn't realize?? Though if the latter, the name should be changed to reflect that.